jira/utils.go

```diff
@@ -39,24 +41,38 @@ func connect(_ context.Context, d *plugin.QueryData) (*jira.Client, error) {
 	if jiraConfig.Token != nil {
 		token = *jiraConfig.Token
 	}
+	if jiraConfig.PersonalAccessToken != nil {
+		personal_access_token = *jiraConfig.PersonalAccessToken
+	}
 
 	if baseUrl == "" {
 		return nil, errors.New("'base_url' must be set in the connection configuration. Edit your connection configuration file and then restart Steampipe")
 	}
-	if username == "" {
+	if username == "" && token != "" {
 		return nil, errors.New("'username' must be set in the connection configuration. Edit your connection configuration file and then restart Steampipe")
 	}
-	if token == "" {
-		return nil, errors.New("'token' must be set in the connection configuration. Edit your connection configuration file and then restart Steampipe")
+	if token == "" && personal_access_token == "" {
+		return nil, errors.New("at least one of 'token' or 'personal_access_token' must be set in the connection configuration. Edit your connection configuration file and then restart Steampipe")
```

Comment:
```suggestion
		return nil, errors.New("'token' or 'personal_access_token' must be set in the connection configuration")
```

I think we can shorten the first part and exclude the last part, since in most cases we don't need to restart Steampipe to re-load connection configs

Can you please update the other error messages as well to be a similar style?